WebApplication/Backend/Services/RegistrationService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Model.Accounts;
+using WebApplication.Backend.Repositorys;
+
+namespace WebApplication.Backend.Services
+{
+    /// <summary>
+    /// This class does connection with repository
+    /// </summary>
+    public class RegistrationService
+    {
+        private IRegistrationRepository iregistrationRepository = new RegistrationRepository();
```

Comment:
the field should be named registrationRepository - i is used only with interface names not fields since the field is a concrete instance of any class that implements this interface